### PR TITLE
Feature/change evaluation mode

### LINKED ISSUE #2: Add documentation
- improve documentation about the action
- teach how to create a development environment

### PR DESCRIPTION
**Pull request by jeanpsv**
close #2

### CODE REVIEW

File: evaluator.js [hunk 1]
```diff
@@ -6,21 +6,36 @@ const WRONG_ANSWER_GRADE = 1;
...
+    }))
+  )).flat()
+    .reduce((acc, evaluation) => {
+      const status = acc[evaluation.describe];
+      const currentStatus = evaluation.status;
+      if (!status || currentStatus === 'failed') {
+        acc[evaluation.describe] = currentStatus;
+        return acc;
+      }
+      if (status === 'failed') return acc;
```
Comment (albertosca) [inline evaluator.js:29]: Isn't this `if` useless here? 🤔

File: tests/jest-output.json [hunk 2]
```diff
@@ -1,54 +1,154 @@
...
-      "ancestorTitles": [
-        "my first describe"
-      ],
-      "fullName": "my third describe my second test",
-      "title": "my second test",
-      "status": "failed"
-    }],
-    "status": "failed"
-  }]
+  "numFailedTestSuites":1,
```
Comment (albertosca) [inline tests/jest-output.json:2]: Are the changes in this file intentional?

**Review by albertosca — CHANGES_REQUESTED** [review]

File: evaluator.js (hunk 1 above)
Comment (jeanpsv) [reply evaluator.js:29]: Yes, You are right. I'll change it

File: tests/jest-output.json (hunk 2 above)
Comment (jeanpsv) [reply tests/jest-output.json:2]: Yes, I update the file to keep entire jest output.

**Review by albertosca — APPROVED** [review]

**Pull request merged**